code/datums/ai_laws.dm

```diff
@@ -223,6 +223,11 @@
 
 /datum/ai_laws/proc/set_laws_config()
 	var/list/law_ids = CONFIG_GET(keyed_list/random_laws)
+
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_UNIQUE_AI))
+		pick_weighted_lawset()
+		return
```

Comment:
multiple admemes liked it but if a headmin dislikes it they can speak up

Reply:
I personally have nothing against random AI lawsets as a trait from a gameplay perspective.

Administratively, I have maximum faith in my fellow admins to be able to enforce silicon policy in light of the fact AI laws are so easy to change to begin with. If an admin struggles to enforce silicon policy properly, it is going to be despite this trait and not because of it.

Players on the other hand are gonna have fun the first time they get a non-Asimov shiftstart AI and it won't open doors for them.